portfolio/src/main/webapp/script.js

```diff
@@ -74,6 +76,12 @@ function createComment(titleText, bodyText, emailText){
     body.appendChild(document.createTextNode(bodyText));
     comment.appendChild(body);
 
+    const score = document.createElement("p");
+    score.className = "sentimentScore";
+    sentimentText = "Sentiment Score: " + sentimentScore.toFixed(2);
+    score.appendChild(document.createTextNode(sentimentText));
+    comment.appendChild(score);
+
```

Comment:
I wouldnt do this in this PR, but you know what would be fun? Coloring the background of this element green or red based on the score.

Reply:
That's a cool idea! I'll start implementing that